src/webgpu/shader/execution/expression/call/builtin/bitcast.spec.ts

```diff
@@ -74,9 +74,18 @@ const f32RangeWithInfAndNaN: number[] = [
   ].map(u => reinterpretU32AsF32(u)),
 ];
 
-const anyF32: Comparator = v => ({ matched: true, got: `${v}`, expected: 'any f32' });
-const anyI32: Comparator = v => ({ matched: true, got: `${v}`, expected: 'any i32' });
-const anyU32: Comparator = v => ({ matched: true, got: `${v}`, expected: 'any u32' });
+const anyF32: Comparator = {
+  impl: v => ({ matched: true, got: `${v}`, expected: 'any f32' }),
```

Comment:
nit: `Comparator` is just TypeScript metadata, and there's no way to know that an object exposing an `impl` field has anything to do with a `Comparator`.
I'd suggest renaming `impl` to `compare` so the field make sense in isolation.

Reply:
Done